tests/drivers/gpio/gpio_nrf/prj.conf

```diff
@@ -1,2 +1,3 @@
 CONFIG_ZTEST=y
 CONFIG_GPIO=y
+CONFIG_GPIO_NRFX_INTERRUPT=n
```

Comment:
It would be good to update `README.txt` accordingly as it only mentions testing of drive strengths.